### ISSUE TITLE
Cannot reproduce the best results from the provided pretrained model

### ISSUE THREAD

**Issue by WinterShiver** [kentonl/e2e-coref#96]
I have successfully loaded the pretrained weights from e2e-coref.tgz and get 72.39 on the test set. According to your NAACL18 paper [paper](https://arxiv.org/abs/1804.05392), this corresponds to "e2e + ELMo + hyperparameter tuning"? I am wondering that how to reproduce the results of "+ c2f inference" and "+ second-order inference"? Could you please provide the corresponding configures of these two settings? Waiting for your apply.
